src/display/mapcanvas.cpp

```diff
@@ -1077,7 +1077,7 @@ void MapCanvas::userPressedEscape(bool /*pressed*/)
 
     case CanvasMouseModeEnum::MOVE:
         // special case for move: right click selects infomarks
-        FALLTHRU;
+        FALLTHROUGH;
```

Comment:
We should extract this out and package it as a separate PR enhancement

Reply:
I could not resist fixing the typo, er, abbreviation :)

Reply:
Reverted. The rename `FALLTHRU` -> `FALLTHROUGH` can be done in a different PR